lisa/tools/ntttcp.py

```diff
@@ -201,6 +201,7 @@ def run_as_server_async(
         process = self.node.execute_async(
             f"ulimit -n 204800 && {self.command} {cmd}", shell=True, sudo=True
         )
+        process.wait_output("threads created")
```

Comment:
Let's use one PR for each fix. So it will be merged earlier.

Reply:
split it into another PR